config/locales/pl/text_msg.pl.yml

```diff
@@ -0,0 +1,5 @@
+pl:
+  pages:
+     text:
+        title: 'Usługa niedostęna w Polsce.'
```

Comment:
This PR is already merged, but I forgot to address something.

I believe the SMS feature *can* be used outside of the U.S. (I haven't tested this personally, but I see no reason why it wouldn't. Again, there might be  international textig fees, if your cell service provider charges for that, but it should still work... There may not be many restrooms in Poland yet, but hopefully there willl be some eventually.)

(Also, the responses are simply an address. See: https://github.com/RefugeRestrooms/refugerestrooms/blob/develop/app/assets/images/text-msg-demo.png)

Could I possibly ask you to submit another PR reverting commit 96ec91e, so the Polish translation for this file goes back to being a full translation (not warning users the service won't work outside the U.S.)?

Or would you mind if I just go ahead and revert that commit in our `develop` branch?

Thanks, and sorry for the oversight in merging before resolving that issue.

Reply:
Actually, As far as I can tell the SMS service does not work at the moment.

I tried texting the number, and it seems to be in use by Lyft. See this issue for details: https://github.com/RefugeRestrooms/refugerestrooms/issues/476

For now this PR is fine as-is. Thanks again for this.